data-structures/records/python/player_records_list.py

```diff
@@ -0,0 +1,78 @@
+# Isaac Computer Science
+# Usage licensed under the Open Government Licence v3.0
+
+
+class PlayerRecord():
+    """Use a class to represent a player as a record"""
+    
+    def __init__ (self):
+        self.player_id = None
+        self.first_name = None
+        self.last_name = None
+        self.date_of_birth = None
+        self.position = None
+        self.injured = None
+
+
+def create_player(p_id, f_name, l_name, dob, pos, inj):
+    """Create a new player record with the given data"""
+    
+    # Create a new player record
+    player = PlayerRecord()
+
+    # Store the details of the player
+    player.player_id = p_id
+    player.first_name = f_name
+    player.last_name = l_name
+    player.date_of_birth = dob
+    player.position = pos
+    player.injured = inj
+
+    # Return the player record
+    return player
+
+
+def display_players(players_list):
+    """Display some of the player details from each player record"""
+    
+    print("### List of player records ###")
+
+    # Repeat for each player in the players list of records
+    for player in players_list:
+        # Display the player's number, name and position
+        print(f"\nNumber: {player.player_id}")
```

Comment:
I was confused by this - is this the player's shirt number or a meaningless ID number? Presumably people can change shirt number? (I don't know much about football)